### PR TITLE
Fix for issue #51

### PR DESCRIPTION
**Pull request by donnapep**
- Check to ensure that Google font has not already been added to the dropdown before adding it again.
- Get jQuery from CDN and use local fallback.
- Single quotes to double quotes.
- Remove test files with no tests in them.
- Remove `proctractor.conf.js` since it doesn't appear to be needed.

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees Please review. Thx.

File: package.json [hunk 1]
```diff
@@ -28,11 +29,9 @@
     "gulp-connect": "~2.0.5",
     "gulp-html2string": "git://github.com/Rise-Vision/gulp-html2string",
     "gulp-minify-css": "^0.3.5",
-    "gulp-protractor": "~0.0.8",
     "gulp-rename": "~1.2.0",
     "gulp-sass": "^0.7.2",
     "gulp-util": "~2.2.17",
-    "mocha": "~1.20.1",
     "phantomjs": "^1.9.7",
     "protractor": "~1.0.0",
```
Comment (stulees) [inline package.json]: @donnapep You should remove the dependency on protractor and rely on widget-tester, thanks

File: bower.json [hunk 2]
```diff
@@ -18,12 +18,16 @@
     "templates"
   ],
   "dependencies": {
-    "angular": "1.2.18",
+    "angular": "~1.3.0",
     "bootstrap-select": "1.5.4",
-    "common-style": "https://github.com/Rise-Vision/common-style.git",
-    "jquery": "1.11.0",
+    "jquery": "~2.1.4",
+    "rv-common-style": "https://github.com/Rise-Vision/common-style.git#1.3.14",
```
Comment (stulees) [inline bower.json]: @donnapep Curious, is upgrading to such a high version going to give any version conflicts in widget-settings-ui-components and our widgets? Will it still be fine in those to override this to be a lower version?

Comment (donnapep) [inline bower.json]: @stulees I think you just need to add a bower resolution for the Widget and it should be fine. Do you think we need to bother with version tags in common repos?

Comment (stulees) [inline bower.json]: @donnapep I don't think so. I'm for leaving the tag off and just take the latest release and let the Widgets specify particular tags.

**Comment by donnapep** [comment]
@stulees I changed to use BDD syntax since that seems to be what other repos use. Guess maybe we'll need to see about switching from TDD to BDD in web components then.

Also made corrections to `bower.json`. Please review. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Pull request merged**